### DISCUSSION TITLE
E0222 917038       1 reflector.go:138] pkg/mod/k8s.io/client-go@v0.20.4/tools/cache/reflector.go:167: Failed to watch *v1beta1.Ingress: failed to list *v1beta1.Ingress: the server could not find the requested resource (get ingresses.extensions)

### DISCUSSION THREAD

**Discussion by aman35** [argoproj/argo-rollouts#1874] [Q&A] [answered]
This is production environment, consisting of Kubernetes, Argo-rollouts and Istio.  Kubernetes -1.22.4 , argo-rollouts -1.1.1 
Everything was working perfectly fine. Then we upgraded Kubernetes to version 1.22.x and everything came backup except argo-rollouts deployment. 
Pod logs shows:

E0222 21:58:34.812148       1 reflector.go:138] pkg/mod/k8s.io/client-go@v0.20.4/tools/cache/reflector.go:167: Failed to watch *v1beta1.Ingress: failed to list *v1beta1.Ingress: the server could not find the requested resource (get ingresses.extensions)
E0222 21:58:37.628680       1 reflector.go:138] pkg/mod/k8s.io/client-go@v0.20.4/tools/cache/reflector.go:167: Failed to watch *v1beta1.Ingress: failed to list *v1beta1.Ingress: the server could not find the requested resource (get ingresses.extensions)
E0222 21:58:41.917038       1 reflector.go:138] pkg/mod/k8s.io/client-go@v0.20.4/tools/cache/reflector.go:167: Failed to watch *v1beta1.Ingress: failed to list *v1beta1.Ingress: the server could not find the requested resource (get ingresses.extensions)

Now I have been through some forums and everyone is suggesting to upgrade argo-rollout to version "v1.2.0". but this is still a pre-release version and we are not allowed to upgrade to pre-release version. 

Is there any other solution to this issue ?

**Comment by harikrongali** [comment, marked as answer]
please use argo rollouts 1.2 version. Support added in 1.2
